artifacts/Directory.Build.props

```diff
@@ -1,3 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why remove these?

Reply:
I'm not sure how they got removed, I didn't touch them, I'll add them back

Reply:
This is weird because the artifacts folder is gitignored, I have no idea how this got deleted

Reply:
Ahh ok. I must have deleted them when I was nuking the artifact folder, I'll be careful with that in the future

Reply:
You're right, we should be able to nuke this folder..

There still seem to be changes in these. You can use `git reset` for resetting individual files back to `main` btw. It's a good way to revert when you're using CLI.